crates/holochain/src/core/workflow/initialize_zomes_workflow.rs

```diff
@@ -117,20 +117,14 @@ where
     };
 
     // Insert the init marker
-    // FIXME: For some reason if we don't spawn here
-    // this future never gets polled again.
     let ws = workspace.clone();
-
-    tokio::task::spawn(async move {
-        ws.source_chain()
-            .put(
-                builder::InitZomesComplete {},
-                None,
-                ChainTopOrdering::Strict,
-            )
-            .await
-    })
-    .await??;
+    ws.source_chain()
```

Comment:
While investigating a flaky test, I found this redundant task. Removed it and everything still works.